p8e-api/src/main/kotlin/io/provenance/engine/service/ChaincodeInvokeService.kt

```diff
@@ -100,9 +98,12 @@ class ChaincodeInvokeService(
                 provenanceGrpc.getLatestBlock()
                     .takeIf { it.block.header.height > currentBlockHeight }
                     ?.let {
-                        log.info("Clearing blockScopeIds")
                         currentBlockHeight = it.block.header.height
-                        blockScopeIds.clear()
+
+                        if (!blockScopeIds.isEmpty()) {
+                            log.info("Clearing blockScopeIds")
```

Comment:
can we set the level to debug? i had to comment this out locally cause it was clogging up the logs.